### DISCUSSION TITLE
I've got issue with User Permissions to create page in another language

### DISCUSSION THREAD

**Discussion by TigerSnow** [sulu/sulu-workshop#40] [Q&A] [answered]
Hi,

I'm sorry for posting but i'm not very comfortable with SULU so I try to practice it.
I'm on the "Pull Request #4", so it's pretty clear for me how to add new localisation, etc...

But now when I'm in the admin panel I cannot create page in another language than english. I've a "No Permissions" and I don't know how to make it works.

Thanks for support and regards ;)

**Comment by alexander-schranz** [comment, marked as answer]
@TigerSnow you mostly skip the step in https://github.com/sulu/sulu-workshop/pull/9:

> Give the admin user permissions for the new languages

I adopted the descripton to add where todo this: `Contacts` -> `Admin` -> `Permissions` -> `Roles` -> Add new language.

**Comment by TigerSnow** [comment]
Oh god... thank you. I feel a little bit stupid, I was convinced that there is something to do in "Settings" > "User Roles" but don't find exactly how it works.

Thank you !

**Reply by alexander-schranz** [reply]
To avoid Roles like "Content Manager EN", "Content Manager DE", "Content Manager IT", ... Sulu did go the way to split this up and save the languages on the `UserRole` relation. This avoids too many roles and on the relation between the User and the Role can be defined for which languages.
